### ISSUE TITLE
[Safari] Secondary subtitles list is not visible

### ISSUE THREAD

**Issue by sh-cho** [gmertes/NflxMultiSubs#23]
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/11611397/163675831-f06fb229-4520-4ba5-9c80-f7fdbdc0d180.png">

It only happens on Safari. Can you check @WingCH ? :)

**Comment by WingCH** [comment]
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/19588517/163677474-bf15eb9a-feda-4663-8da6-245601d79931.png">
 i am work normally.
 
Can get me your netflix link?

**Comment by sh-cho** [comment]
https://www.netflix.com/watch/81488262?trackId=254015180&tctx=0%2C0%2C1d5bb5a3-0672-4220-a4af-8b71ab254824-227645080%2C8c9bf957-000b-4914-b831-764e7944a72b_119804429X20XX1650113595374%2C8c9bf957-000b-4914-b831-764e7944a72b_ROOT%2C%2C%2C

**Comment by WingCH** [comment]
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/19588517/163695907-adff300e-b9d2-480a-ac38-c860864fe9f7.png">

cannot reproduce..
i am work normally.

**Comment by gmertes** [comment]
@sh-cho can you post the log? Refreshing once after starting playback doesn't fix it? One possible reason this can happen is if you don't let the nflx home page load completely before starting a show. Also hovering over the show tile for a few seconds before clicking it can sometimes fix this.

**Comment by sh-cho** [comment]
```
[Log] NflxMultiSubs> – "Injected: nflxmultisubs.min.js" (content.min.js, line 1, x2)
[Log] NflxMultiSubs> – "Version: 2.2.9" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Version: 2.2.9" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Intecerpted manifest: 81309354" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Intecerpted manifest: 81309354" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Activating manifest 81309354 (last=undefined)" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Language mode: " – "audio" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Default audio track language: en" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Subtitle #1 auto-enabled to match audio" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Subtitle \"영어 [CC]\" downloading" (nflxmultisubs.min.js, line 2)
[Debug] NflxMultiSubs> – "Selecting fastest server, candidates: " – ["https://ipv4-c013-hkg001", "https://ipv4-c020-pdx001", "https://ipv4-c035-pdx001"] (3) (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Started: renderer loop" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "hasNeoStyleControls: false" (nflxmultisubs.min.js, line 2)
[Debug] NflxMultiSubs> – "Fastest: https://ipv4-c035-pdx001" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Subtitle \"영어 [CC]\" loaded" (nflxmultisubs.min.js, line 2)
```

<img width="1748" alt="image" src="https://user-images.githubusercontent.com/11611397/164967351-743ffda1-7fba-4b36-9c68-2521e1db0c02.png">

@gmertes 

Subtitles list is still invisible but also somehow secondary subtitles is loaded at the same time.

Refreshing or other methods are not working so far 😒

**Comment by WingCH** [comment]
It seems to be some strange problems
Have you tried reinstalling the plugin?

**Comment by sh-cho** [comment]
Yep still the same

**Comment by gmertes** [comment]
```
[Log] NflxMultiSubs> – "Injected: nflxmultisubs.min.js" (content.min.js, line 1, x2)
[Log] NflxMultiSubs> – "Version: 2.2.9" (nflxmultisubs.min.js, line 2)
[Log] NflxMultiSubs> – "Version: 2.2.9" (nflxmultisubs.min.js, line 2)
```

From these lines it looks as if the script is being injected 2 times. Rest of the log is normal. If the script is in fact loaded 2 times, that could cause conflicts in the sub list injection.

**Comment by gmertes** [comment]
> This issue is happening on chrome now as well.

Uh oh. Likely that something else broke unrelated to this Safari issue. I'll investigate but I may need a few days.

**Comment by gmertes** [comment]
So, uh, any updates on this @sh-cho @WingCH ? 😅

**Comment by WingCH** [comment]
<img width="1548" alt="image" src="https://user-images.githubusercontent.com/19588517/218314833-d6931723-1718-4c99-9c3e-c0c05302b1dd.png">

I've tried to reproduce the issue on my macbook but I wasn't able to do so. 
I am using the [NflxMultiSubs App store version 2.4.3](https://apps.apple.com/hk/app/nflxmultisubs/id1594059167?mt=12) and Safari version 16.2 (18614.3.7.1.5).  😵

**Comment by sh-cho** [comment]
@gmertes Hi, I forgot to reply. (sorry)

<img width="1384" alt="image" src="https://user-images.githubusercontent.com/11611397/219871779-28b0338d-483c-4c71-a3f8-7300385497e8.png">

Unfortunately, it is still happening on me.. 😞 don't know why.

Also, since I don't know when, I don't see any secondary subtitles now. 


<img width="459" alt="image" src="https://user-images.githubusercontent.com/11611397/219871969-c53bd221-451c-4340-93be-149cd787d4e0.png">

<img width="454" alt="image" src="https://user-images.githubusercontent.com/11611397/219871995-f31e4f16-45c8-48b7-aa25-079987484e0f.png">

This is safari console log.

**Comment by WingCH** [comment]
@sh-cho what is your Safari version?

**Comment by sh-cho** [comment]
@WingCH 
`Version 16.3 (18614.4.6.1.5)`

**Comment by WingCH** [comment]
I'm having trouble finding a download link for Safari Beta (version 16.3), but I was able to successfully download Safari Technology Preview (version 16.4). I haven't encountered any issues with version 16.4 so far. 🫣

<img width="1360" alt="image" src="https://user-images.githubusercontent.com/19588517/219955939-ef6379c9-df59-40c8-833c-2954e8ff415e.png">

**Comment by sh-cho** [comment]
hmm ok.. it's really weird 🥲 

I'm gonna fix it by myself some day.

**Comment by WingCH** [comment]
Have you tried reinstalling Safari and extension?

**Comment by kazdrubal** [comment]
I also have this problem on Firefox in Oct. 2023, so I came here to see if there is a solution. But it seems that other browser extensions are blocking MultiSubs from working. It's worth investigating.

**Comment by gmertes** [comment]
Hey @kazdrubal , I can't reproduce any issues on latest Firefox - everything is working here. Can you try disabling all other extensions and trying again? If it still doesn't work, a console log would be helpful. You can create a new issue for it.

**Comment by kazdrubal** [comment]
I figured it out, the issue is caused by the 'Video Speed Controller' extension! When disabled, 'NflxMultiSubs' works just fine! I think this is important for other users to know who might have the same problem, to first disable extensions.
